src/engine/BMInterfaceGame.php

```diff
@@ -49,23 +48,43 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                if ($this->isTest) {
```

Comment:
So, change all the multiple calls to create_game() in the test code? I believe that there are many of these.